src/routes/project/TaskDescription.svelte

```diff
@@ -3,27 +3,94 @@
 	import Backdrop from '$lib/components/Backdrop.svelte'
 	import Panel from '$lib/components/Panel.svelte'
 	import { escape } from '$lib/directives/escape'
+	import { Character } from '$lib/model/character'
+	import WaitingImage from '$lib/components/WaitingImage.svelte'
 	export let isOpen: boolean = false
 	export let task: Task | null
 	export let close: VoidFunction
+	export let assignee: Character | undefined
 </script>
 
 <Backdrop {isOpen}>
 	<Panel>
 		<div class="container" use:escape on:escape={close}>
-			<h3 class="title">{task?.name}</h3>
+			<h3>{task?.name}</h3>
 			<div class="description">{task?.description}</div>
-			<div class="additional-data">
-				<p class="status">Status: {task?.status}</p>
-				<p class="estimate">Estimated time: {task?.estimate}</p>
+			<div class="resume">
+				<div class="resume__statuses">
+					<p class="resume__status">
+						<span class="resume__title">Status:</span>
+						<span class="resume__value">{task?.status}</span>
+					</p>
+					<p class="resume__status">
+						<span class="resume__title">Estimated time:</span>
+						<span class="resume__value">{task?.estimate}</span>
+					</p>
+					{#if task?.status === 'done'}
+						<p class="resume__status">
+							<span class="resume__title">Spent time:</span>
+							<span class="resume__value">{task.timeSpent.toFixed(2)}</span>
+						</p>
+					{/if}
+				</div>
+				<div class="resume__status resume__status--separated">
+					<span class="resume__title">Assigned to:</span>
+					{#if assignee}
+						<div class="resume__assignee">
+							<div class="resume__avatar">
+								<WaitingImage src={assignee.image} alt={assignee.id} width={75} height={75} />
+							</div>
+							<div class="assignee__name">{assignee?.name}</div>
+						</div>
+					{/if}
+					{#if task?.status === 'todo'}
+						<slot />
+					{/if}
+				</div>
 			</div>
 		</div>
 	</Panel>
 </Backdrop>
 
 <style>
 	.container {
-		width: max-content;
+		min-width: 50dvw;
 		margin: 0 auto;
 	}
+	.description {
+		margin-bottom: 2rem;
+	}
+	.resume__status {
+		margin: 0.5rem;
+	}
+	.resume__status--separated {
+		margin-top: 2rem;
+	}
+	.resume__title {
+		font-weight: bold;
+	}
+	.resume__value {
+		text-transform: uppercase;
```

Comment:
Why do you so love to shout at users?? 🤣  The purpose of uppercase is to grab user's attention. Usually used in e.g. call-to-action, main title etc. And.. well, if you have a bad font. Or writing SQL queries. And it is usually should be the only element on the page. Why would you have _all values_ in uppercase?